docs/Scaling compute/Polly CLI.md

```diff
@@ -175,6 +175,91 @@ This command will copy an individual file from source to destination. The transf
 
 *   If you are within a Polly environment while accessing Polly CLI (like notebook or CLI job), workspace id can be auto-detected by passing --yes or -y in the command instead of --workspace-id parameter.
 
+##Docker Management
+
+Polly has its own docker repository where dockers can be managed. The advantage of the Polly Docker Repository is that these dockers can also be stored within the same platform as data, code, and analyses and to access the stored dockers, all you need is the Polly login.
```

Comment:
"and to access the stored dockers, all you need is the Polly login." rephrase to "and to access the stored dockers all you need is the Polly login credential."